WordPress/gutenberg
# Social Links: update Facebook's color to match brand guidelines

## What?

This PR updates Facebook's icon color in the Social Links block, to match the brand guidelines defined by Meta.

See the guidelines defined here:
https://about.meta.com/brand/resources/facebook/logo/

## Why?

This change updates the Facebook icon to match other Facebook representations one can find online.

## How?

| **Before** | **After** |
|--------|--------|
| <img width="71" alt="Screenshot 2024-04-03 at 14 57 00" src="https://github.com/WordPress/gutenberg/assets/426388/8c5f69f8-fcdf-468e-84d3-4f6b4782eeea"> | <img width="102" alt="Screenshot 2024-04-03 at 14 56 34" src="https://github.com/WordPress/gutenberg/assets/426388/d28d90b5-9df1-4234-a712-e17d9e48c945"> | 

## Testing Instructions

1. Open the site editor
2. Insert a Social Links block
3. Insert a Facebook icon, and give it a URL.
4. Save your changes.
5. The Facebook icon you see on the frontend should match the icon seen on [this page](https://about.meta.com/brand/resources/facebook/logo/).

### Testing Instructions for Keyboard

No changes here.

## Screenshots or screencast

See above
